src/test/java/build/buildfarm/worker/StubWorkerContext.java

```diff
@@ -207,4 +207,9 @@ public IOResource limitExecution(
   public int commandExecutionClaims(Command command) {
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  public boolean shouldErrorOperationOnRemainingResources() {
+    throw new UnsupportedOperationException();
```

Comment:
This call happens for every execution, whether the IOResource is meaningful or not. Have it return the configured value, as you've added it to the operationqueue worker config definition.

Reply:
Apologies, I misread the context and thought this was the operationqueue worker